README.md

```diff
@@ -15,7 +72,342 @@ def deps do
 end
 ```
 
-Documentation can be generated with [ExDoc](https://github.com/elixir-lang/ex_doc)
-and published on [HexDocs](https://hexdocs.pm). Once published, the docs can
-be found at [https://hexdocs.pm/colonel_kurtz_ex](https://hexdocs.pm/colonel_kurtz_ex).
+Documentation can be generated with [ExDoc](https://github.com/elixir-lang/ex_doc) and published on [HexDocs](https://hexdocs.pm). Once published, the docs can be found at [https://hexdocs.pm/colonel_kurtz_ex](https://hexdocs.pm/colonel_kurtz_ex).
+
+## Getting Set Up
+
+To get set up with `ColonelKurtzEx`, you'll need to install and configure [Colonel Kurtz](https://github.com/vigetlabs/colonel-kurtz). Since the two libraries go hand in hand, you'll often jump back and forth between the two. For example, when you add a new block type to `CKJS`, you'll need to add the corresponding modules for `CKEX` (a `BlockType`, `BlockTypeView`, and template).
+
+**Note**: The following sections are expandable (and are collapsed by default).
+
+<details>
+  <summary>
+    <strong>1. Add Folders for BlockTypes and BlockTypeViews</strong>
+  </summary>
+
+  After adding the library to your dependencies, you'll want to define a few modules in the scope of your application. One for your custom `BlockType` definitions, and one for your `BlockTypeView`s.
+
+  **Note:** *it is important that each of these concepts live inside some module namespace in your application so that the library can look up specific block type and view modules at runtime.*
+
+<details>
+  <summary>
+    See an example
+  </summary>
+
+  For example, assuming a standard phoenix project structure:
+
+  1. Create a new subfolder inside `lib/your_app_web/views/`, using whatever name you'd like that corresponds with the module you'll be defining views inside (e.g. `lib/your_app_web/blocks/` folder and `YourAppWeb.Blocks` module.).
+
+  2. Create a new subfolder inside `lib/your_app/`. Again, the name doesn't matter so long as you configure `ColonelKurtzEx` correctly (more information in the following section). For example, you might creat a folder named `lib/your_app/block_types/` and to contain the `YourApp.BlockTypes` namespace.
+</details>
```

Comment:
I recognized a typo here after the fact (missing `views` in one of those example paths). I was endeavoring to provide consistent examples.

There are 2 separate namespaces you define in your app: 1 for views and 1 for block types. In the examples listed above I am putting block types into the non `web` folder (i.e. `lib/your_app/block_types`) because they're a data concern. I'm putting views in a subfolder of the `web` folder (i.e. `lib/your_app_web/views/blocks/`).